packages/cli/src/templates/project/package-json.ts

```diff
@@ -26,9 +26,9 @@ export const template = `{
     "@types/mocha": "8.2.2",
     "nyc": "15.1.0",
     "prettier":  "2.3.0",
-    "typescript": "4.5.4",
+    "typescript": "4.7.4",
     "ts-node": "9.1.1",
-    "@types/node": "15.0.2",
+    "@types/node": "18.0.6",
```

Comment:
Shouldnt this be 16?

Reply:
Not sure about it. ./package.json is `"@types/node": "16.11.7"` but in the integration-folder `package.json` there is not `@types/node`

Reply:
Done